CSLabs.Api/Models/Maintenance.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace CSLabs.Api.Models
+{
+    public class Maintenance
+    {
+        public int Id { get; set; }
+        public bool IsMaintenanceMode { get; set; }
```

Comment:
If this database row represents a maintenance then isn't is already in maintenance mode making this property redundant?
```suggestion
```

Reply:
The idea was that whenever the clusters are down we would check whether we are in maintenance mode to change the information shown on the 503 page. So if the servers are down then we check if it's in a maintenance, and if not, then we just say that our servers are down. If it is, then we tell the users that there is currently maintenance and tell them approximately when it will be back up.

Reply:
I agree with jason